src/ergw_gtp_c_socket.erl

```diff
@@ -453,7 +453,7 @@ handle_err_input(Socket, State) ->
     State.
 
 handle_message(ArrivalTS, Src, IP, Port, Data, #state{gtp_socket = Socket} = State0) ->
-    try gtp_packet:decode(Data, #{ies => binary}) of
+    try gtp_packet:decode(Data, #{ies => map}) of
```

Comment:
decoding to a map is too soon. This should only happen in the handler process, not in the socket process